packages/terra-application-navigation/src/tabs/Tab.module.scss

```diff
@@ -99,7 +99,7 @@
     &[aria-current='false']:active {
       .tab-inner {
         background-color: var(--terra-application-navigation-tab-active-background-color, rgba(3, 43, 70, 0.4));
-        color: var(--terra-application-navigation-tab-active-background-color, rgba(255, 255, 255, 0.5));
+        color: var(--terra-application-navigation-tab-active-color, rgba(255, 255, 255, 0.5));
```

Comment:
This should have been flagged by the duplicate property rule. The rgba value might return an ast that needs special logic.